src/Test.UnitTests.Sarif.Driver/Sdk/ExportConfigurationCommandBaseTests.cs

```diff
@@ -62,12 +64,8 @@ public void ExportConfigurationCommandBase_VerifyDefaultConfiguration()
 
             configuration.Should().NotBeNull();
 
-            // We only have a single rule that provides options. Every rule
-            // provides a default setting, however, that controls whether 
-            // a rule is enabled or forced to a particular warning level.
-            // We therefore expect two top-level items in the configuration,
-            // each of which is a properties collection for a rule.
-            configuration.Keys.Count.Should().Be(3);
+            // We export two rules, FunctionlessTestRule and TestRule
```

Comment:
>// [](start = 12, length = 2)

Ultra-nit: Now that you have the self-explanatory, self-maintaining helper function `NumberOfExportedRules`, you don't need these two comments, which might get out of date.